src/picked_protein/Protein.cpp

```diff
@@ -640,8 +639,8 @@ void Protein::setSequence(
   )
 {
 
-  free(sequence_);
   unsigned int sequence_length = strlen(sequence) +1; // +\0
+  free(sequence_);
```

Comment:
Was this change done for a reason? Not important, just wondering.

Reply:
I probably thought sequence_ and sequence were the same variable, but now I see they aren't.